.circleci/config.yml

```diff
@@ -55,7 +55,12 @@ jobs:
           paths:
             - node_modules
       - run:
-          command: yarn download && GATSBY_CPU_COUNT=1 yarn build
+          environment:
+            ENABLE_SENTRY: 'true'
+            SENTRY_NO_PROGRESS_BAR: '1'
```

Comment:
Where will `SENTRY_NO_PROGRESS_BAR` be consumed?

Reply:
Used by sentry webpack plugin (via [sentry-cli](https://docs.sentry.io/product/cli/configuration/) ) so that it won't litter the `yarn build` output by displaying a progress bar.